api/namex/models/request.py

```diff
@@ -129,24 +129,11 @@ class Request(db.Model):
     # properties
     @property
     def has_consumed_name(self):
-        names = self.names.all()
-
-        def is_not_consumed(name):
-            no_corp_num = name.corpNum is None
-            no_consumption_date = name.consumptionDate is None
-
-            return no_corp_num and no_consumption_date
-
-        results = [is_not_consumed(n) is False for n in names]
-        return True in results
+        return self.stateCd == State.CONSUMED
 
     @property
     def is_expired(self):
-        todays_date = datetime.utcnow().date()
-        expiry_date = self.expirationDate.date() if self.expirationDate else None
-        if expiry_date:
-            return todays_date > expiry_date
-        return False
+        return self.stateCd == State.EXPIRED
```

Comment:
This is relying on the job to set EXPIRED, correct?
If so, would it not be safer to:
```suggestion
if State.EXPIRED or \
   self.expirationDate and datetime.utcnow().date() > self.expirationDate.date():
    return True
return False
```

Reply:
I changed it here. I assume this won't happen since we have the job running. If the state did not change the UI's will be broken as its depending on EXPIRED status.